### PR TITLE
feat: Getting started screen with user account validation via email

### PR DESCRIPTION
**Pull request by dilumdesilva**
Implemented: #20 

Changes: 
Implemented the getting started screen which asks the user for the email to validate whether the
user already has an account or not.
if the user has an account already the starting screen will navigate directly to the login screen
after the email validation. if the user doesn't have an account it will redirect the user to the signup screen.

Screenshots for the change: 
Screen Shot 2019-04-07 at 1 33 27 PM

### CODE REVIEW

File: EventyayOrganizer/Controllers/SignUpController/SignUpVCMethods.swift [hunk 1]
```diff
@@ -53,8 +53,13 @@ extension SignUpViewController {
         addressTextField.placeholderActiveColor = .defaultColor()
         addressTextField.dividerNormalColor = .iOSGray()
         addressTextField.dividerActiveColor = .defaultColor()
+        addressTextField.text = "https://api.eventyay.com/v1/"
```
Comment (chashmeetsingh) [inline EventyayOrganizer/Controllers/SignUpController/SignUpVCMethods.swift]: Add all constants in a constants file.

File: EventyayOrganizer/Network/APIClient.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+//
+
+import Foundation
+import Alamofire
+import AlamofireObjectMapper
+import ObjectMapper
+
+class APIClient {
+    
+   var kBaseURL = "https://open-event-api-dev.herokuapp.com/v1"
```
Comment (chashmeetsingh) [inline EventyayOrganizer/Network/APIClient.swift]: Same here.

**Review by chashmeetsingh — CHANGES_REQUESTED** [review]
Move all the constants to a constants file, refer SUSI_iOS repo. Looks good otherwise.

**Comment by dilumdesilva** [comment]
@chashmeetsingh sure I will add a constants file and move all the URLs and relevant data there.

**Comment by dilumdesilva** [comment]
@chashmeetsingh could you please have a look into this #29 PR since it includes some relevant changes to this PR.

**Comment by chashmeetsingh** [comment]
Please fix the lint issues.

**Comment by dilumdesilva** [comment]
Oh sure I will fix them.

**Review by chashmeetsingh — APPROVED** [review]
LGTM

**Review by jogendra — APPROVED** [review]

**Pull request merged**